.gitlab-ci.yml

```diff
@@ -0,0 +1,15 @@
+image: ubuntu:bionic
+
+variables:
+  GRADLE_VERSION: "4.10.3"
+
+before_script:
+  - apt-get update && apt-get -y upgrade
+  - apt-get -y install openjdk-11-jdk gradle
+  - gradle -c empty-settings.gradle wrapper --gradle-version=$GRADLE_VERSION --info --stacktrace
+  - chmod +x ./gradlew
+
+build:
+  script:
+    - java -version
+    - LC_ALL=C.UTF-8 ./gradlew build --info --stacktrace
```

Comment:
I think we should remove the `--info` from here and above; it creates very long output that GitLab can only hardly handle.

Reply:
Agree, once everything else is debugged we should definitely remove it.

Reply:
OK, everything seems to be working -- both locally with Docker/gitlab-runner on macOS and on hosted Gitlab. So I've removed `--info`.